### PR TITLE
Bump MappedArrays, TestImages, ImageCore

### PR DESCRIPTION
**Pull request by timholy**

### CODE REVIEW

File: TestImages/versions/0.1.2/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+Images 0.5
```
Comment (tkelman) [inline TestImages/versions/0.1.2/requires]: should this have been changed to FileIO ?

Comment (timholy) [inline TestImages/versions/0.1.2/requires]: Thanks.

Comment (tkelman) [inline TestImages/versions/0.1.2/requires]: https://github.com/JuliaImages/TestImages.jl/releases too or travis will (at least should) fail

Comment (tkelman) [inline TestImages/versions/0.1.2/requires]: oh nevermind not if you didn't change the sha, this should be fine

**Pull request merged**